openapi.yml

```diff
@@ -6373,6 +6314,97 @@ paths:
         name: affects__flaw__source
         schema:
           type: string
+          enum:
+          - ''
```

Comment:
does empty mean None (or null, or ...)  ... guessing this is just reflecting reality of data we got from BZ